assets/scss/_footer.scss

```diff
@@ -0,0 +1,82 @@
+@use 'uswds-core' as *;
+
+$widescreen: 1400px;
+
+.footer {
+  @include u-padding(0);
+  background-color: color('primary-lightest');
+}
+
+.footer-navigation, .usa-identifier__inner {
+  max-width: $widescreen;
```

Comment:
I would have expected the grid styles would be able to handle this for us.

I was tinkering a bit locally, and this diff seems to work?

```diff
diff --git a/_includes/footer-navigation.html b/_includes/footer-navigation.html
index 9f76150..0616f87 100644
--- a/_includes/footer-navigation.html
+++ b/_includes/footer-navigation.html
@@ -1,2 +1,2 @@
-<div class="footer-navigation">
+<div class="footer-navigation grid-container">
   <div class="footer-navigation__section margin-y-2">
diff --git a/_includes/footer.html b/_includes/footer.html
index 9052c9c..4d7fa34 100644
--- a/_includes/footer.html
+++ b/_includes/footer.html
@@ -3,3 +3,2 @@
   <div class="usa-identifier">
-    <div class="usa-identifier__inner">
       <section class="usa-identifier__section usa-identifier__section--masthead"
@@ -78,3 +77,2 @@
       </section>
-    </div>
   </div>
diff --git a/assets/scss/_footer.scss b/assets/scss/_footer.scss
index be7199e..7dc8aa4 100644
--- a/assets/scss/_footer.scss
+++ b/assets/scss/_footer.scss
@@ -2,4 +2,2 @@
 
-$widescreen: 1400px;
-
 .footer {
@@ -9,13 +7,2 @@ $widescreen: 1400px;
 
-.footer-navigation, .usa-identifier__inner {
-  max-width: $widescreen;
-
-  @include at-media('widescreen') {
-    margin: {
-      left: auto;
-      right: auto;
-    }
-  }
-}
-
 .footer-navigation {
@@ -56,15 +43,2 @@ $widescreen: 1400px;
 
-.usa-identifier__inner {
-  display: flex;
-  flex-wrap: wrap;
-}
-
-.usa-identifier__section {
-  flex: 1 1 auto;
-}
-
-.usa-identifier__section--masthead, .usa-identifier__section--usagov {
-  flex-grow: 0;
-}
-
 .system-status {
```

Reply:
Thanks, that's a lot better, I didn't like that `$widescreen` variable in there.
I'm still getting used to the USWDS Grid System.

Reply:
>I'm still getting used to the USWDS Grid System.

Me too 😄